src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -254,6 +254,38 @@ public static string LocalSlotName
                 "{instanceId}: Function '{functionName} ({functionType})', version '{version}' received a '{eventName}' event. State: {state}. HubName: {hubName}. AppName: {appName}. SlotName: {slotName}. ExtensionVersion: {extensionVersion}. SequenceNumber: {sequenceNumber}.",
                 instanceId, functionName, functionType, version, eventName, FunctionState.ExternalEventRaised, hubName, LocalAppName, LocalSlotName, ExtensionVersion, this.sequenceNumber++);
         }
+        public void SendMessageFailed(
```

Comment:
Let's name this `SendNotificationFailed`

Reply:
Sure. We will fix it. Thank you!